packages/forms-web-app/src/controllers/register/myself/full-name.js

```diff
@@ -1,25 +1,46 @@
 const { VIEW } = require('../../../lib/views');
 
+function sanitiseFormPostResponse(error, url) {
```

Comment:
Might want to pull this out of here as it's replicated in 4 places, just wasn't sure where to put it and it's only 3 lines.

Reply:
perhaps `src/utils`?

also, using arrow function syntax would be preferable imo
```
const sanitiseFormPostResponse = (error, url) => {
    // implementation
};
```

Reply:
i'm not actually clear what that function is doing. The name implies it is sanitising something, but just assigns the params to `this` (not sure what `this` refers to in this context).

Reply:
OK i looked a bit more carefully and think i understand, it's a constructor. So arrow function not appropriate!

Reply:
Good shout, I have moved this to utils and replaced the 8 or so instances with an import.